lib/date-utils.js

```diff
@@ -7,10 +7,13 @@ import dayjs from './dayjs';
  * For a given date, return the next charge date.
  *
  * @param {Date} firstChargeDate
- * @param {month|year} interval
+ * @param {week|month|year} interval
  */
 export const getNextChargeDate = (firstChargeDate, interval) => {
-  if (interval === INTERVALS.month) {
+  if (interval === INTERVALS.week) {
+    // Add 1 week (604800000 milliseconds) to firstChargeDate
+    return new Date(firstChargeDate.getTime() + 604800000);
```

Comment:
Re-posting my question from the previous PR: For monthly contributions, we only charge on the 1st of the month. More info about that here: https://docs.opencollective.com/help/financial-contributors/payments#when-will-i-be-billed-next-time.

Now for weekly contributions, I wonder whether we want the same mechanism (e.g. charge every Monday) or if we should go something similar to what we do with Paypal (i.e. nextChargeDate = today + 7 days). @znarf what do you think?

Reply:
"today + 7 days" sounds good. Let's try to make it work that way.